fe/fe-core/src/main/java/com/starrocks/qe/DdlExecutor.java

```diff
@@ -230,6 +233,12 @@ public static void execute(Catalog catalog, DdlStmt ddlStmt) throws Exception {
             catalog.getAnalyzeManager().removeAnalyzeJob(((DropAnalyzeJobStmt) ddlStmt).getId());
         } else if (ddlStmt instanceof RefreshExternalTableStmt) {
             catalog.refreshExternalTable((RefreshExternalTableStmt) ddlStmt);
+        } else if (ddlStmt instanceof CreateWorkGroupStmt) {
+            // TODO: Not implemented
```

Comment:
throw now implemented exception will be better?

Reply:
needless, this pr is one of pr series into which a big pr is parted.